go.mod

```diff
@@ -9,6 +9,8 @@ require (
 	github.com/gosuri/uitable v0.0.4
 	github.com/kudobuilder/kuttl v0.1.0
 	github.com/manifoldco/promptui v0.6.0
+	github.com/onsi/ginkgo v1.11.0
+	github.com/onsi/gomega v1.8.1
```

Comment:
I thought we previously had these as dependencies and they were removed... now they are back?
I'm not against...  but now find myself curious if there are competing mental models

Reply:
Unit tests do not need it and neither does kuttl. However, `instance_admission_integration_test` is complex enough that ginkgo becomes necessary. Gomega is not necessary but both seem to be used together everwhere else so I didn't want to break that pattern 🤷‍♂